File: web/src/app/modules/game-data/model/game.ts
import {Round} from './round';
import {Team} from './team';

export class Game {
  teams: Team[] = undefined;
  rounds: Round[] = undefined;

  public static parseGame(input: any): Game {
    const json = JSON.parse(input);
    return {
      teams: json['teams'],
      rounds: json['rounds']
    };
  }

  constructor(teams: Team[], rounds: Round[]) {
    this.teams = teams;
    this.rounds = rounds;
  }
}

File: core/src/main/java/eu/kotkas/villak/core/roosid/model/Game.java
package eu.kotkas.villak.core.roosid.model;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Game implements Serializable {
    private static final long serialVersionUID = 5911818142075747766L;
    private List<Team> teams;
    private List<Round> rounds;
    private List<Message> latestMessages;
}
